### DISCUSSION TITLE
Welcome to PankuConsole Discussions!

### DISCUSSION THREAD

**Discussion by Ark2000** [Ark2000/PankuConsole#19] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Porfik** [comment]
Has anyone created a fork for version 3.5?

**Reply by Ark2000** [reply]
it's in plan now.

**Comment by ArmoredBear** [comment]
Hi, does this addon work with C#? Thanks

**Reply by Ark2000** [reply]
yes, since they all share the same underlying godot scripting api.
